build.py

```diff
@@ -142,7 +142,7 @@ def do_make(target=None, log=False):
 token = None
 job = None
 boot_cmd = None
-use_git = False
+use_git = True
```

Comment:
I'll update to set use_git = False when -e is set.

While I could just add -g, I have scripts/tooling around build.py that don't use -g, and those broke when the use_git was added, so I consider this a regression since previous behavior was always to use git.